frontend/lib/auth/store.ts

```
@@ -0,0 +1,304 @@
+      setSession: (session) => set({ session }),
+      setLoading: (loading) => set({ loading }),
+      setInitialized: (initialized) => set({ initialized }),
+      setAuthSubscription: (subscription) => set({ authSubscription: subscription }),
+
+      signInWithPassword: async (email: string, password: string) => {
+        set({ loading: true })
+        try {
+          await serverSignInWithPassword(email, password)
+          // The session will be updated through the auth listener
+        } catch (error: unknown) {
+          throw error
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the catch clause.


(lint/complexity/noUselessCatch)